pkg/sso/client.go

```diff
@@ -74,6 +75,9 @@ type GetAuthorizationURLOptions struct {
 	// The app/company domain without without protocol (eg. example.com).
 	Domain string
 
+	// Authentication service provider descriptor.
```

Comment:
We should add in the documentation that this is currently used only for Google as a provider.